storage/tianmu/loader/parsing_strategy.cpp

```diff
@@ -113,15 +113,23 @@ inline bool ParsingStrategy::SearchUnescapedPattern(const char *&ptr, const char
     while (ptr < search_end && *ptr != *c_pattern) {
       if (escape_char_ && *ptr == escape_char_)
         ptr += 2;
-      else
+      else if (string_qualifier_ && *ptr == string_qualifier_ && ptr + 1 < search_end &&
+               *(ptr + 1) == string_qualifier_) {
+        // continuous double enclosed char is reterpreted as a general char.
+        // ref:https://dev.mysql.com/doc/refman/5.7/en/load-data.html
+        ptr += 2;
+      } else
         ++ptr;
     }
   } else if (size == 2) {
     --search_end;
     while (ptr < search_end && (*ptr != *c_pattern || ptr[1] != c_pattern[1])) {
       if (escape_char_ && *ptr == escape_char_)
         ptr += 2;
-      else
+      else if (string_qualifier_ && *ptr == string_qualifier_ && ptr + 1 < search_end &&
```

Comment:
this piece of code is repeated 4 times, how about add a new function for it?
----
(string_qualifier_ && *ptr == string_qualifier_ && ptr + 1 < search_end &&
               *(ptr + 1) == string_qualifier_)

Reply:
fixed it.